gpt_researcher/retrievers/pubmed_central/pubmed_central.py

```
@@ -1,174 +1,152 @@
     """
-    PubMed Central API Retriever
+    PubMed Central Full-Text Search
     """
 
-    def __init__(self, query, query_domains=None):
-        """
-        Initializes the PubMedCentralSearch object.
-        Args:
-            query: The search query.
-        """
+    def __init__(self, query: str, query_domains=None):
```

Ruff (0.14.3)

Unused method argument: `query_domains`

(ARG002)